### PR TITLE
Adds blocked_inbox_or_channels to ProfileWithoutEmail

### PR DESCRIPTION
**Pull request by cloudify**

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please include a Pivotal story at the beginning of the PR title (see below).
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  Please include a description of your PR changes.
  </td>
    </tr>
  </tbody>
</table>



  Example of PR titles that include pivotal stories:

  * single story: `[#123456] my PR title`
  * multiple stories: `[#123456,#123457,#123458] my PR title`

    
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: api_proxy.yaml [hunk 1]
```diff
@@ -493,8 +493,6 @@ definitions:
     title: Profile with wmail
     description: Describes the user's profile.
     properties:
-      blocked_inbox_or_channels:
```
Comment (lussoluca) [inline api_proxy.yaml]: why this move? previously were alphabetically sorted

**Comment by gunzip** [comment]
how does this work with #322 ?

**Comment by cloudify** [comment]
Superseded by PR #324

**Pull request closed without merging**